### ISSUE TITLE
Did somebody change NBG init?

### ISSUE THREAD

**Issue by DanielNScott** [EDmodel/ED2#143]
It used to be the case that restricting INCLUDE_THESE_PFTS to one type and running an NBG simulation would give you only one cohort... it seems like somebody changed this?

Or am I just mis-remembering?

**Comment by mpaiao** [comment]
It should start with a single cohort, but then the vegetation dynamics should create more cohorts as they cohorts grow and reproduce.   If you want to keep this single cohort forever and you don't mind it will remain small, you can also turn of vegetation dynamics (IVEGT_DYNAMICS=0).

**Comment by DanielNScott** [comment]
Yeah that was what I thought (and was used to) but don't believe it does anymore.

**Comment by DanielNScott** [comment]
Ok, I just double checked, it does in the mainline but for some reason in one of my branches it does not. Unsure why, since I don't recall ever having made any intentional changes to that.

**Comment by DanielNScott** [comment]
Aha! Ok, I ran into this again and it seems like INCLUDE_THESE_PFTS get's ignored when there's a config.xml present. Is this supposed to be the case?

**Comment by mdietze** [comment]
That sounds vaguely familiar (though it would have been written ~8yr ago). I may have wanted to avoid inconsistencies between INCLUDE_THESE_PFTS and the PFTs in the xml, in which case adding the PFT to the xml with no parameters (just the ID number) should work to include them. If not, feel free to modify.

**Comment by fabeit** [comment]
For me it doesn't work like that, it seems to work in an additive manner. If you include pft 2,3 and then your xml has 5,6 (even just the numbers with no parameters), then your run will have 2,3,5,6. Personally I would want that if an xml is present, ONLY the pfts in the xml are included, and the ones in ed2in ignored.

**Comment by mdietze** [comment]
That's a bit counter to the original design, which was that the XML is optional and only changed existing PFTs. If you went this route the XML would become required to do a default run and you could end up with a lot of PFTs that are just the number.

I didn't remember that the inclusion of PFTs in an XML makes them active -- that probably should be disabled.
